doc/first-steps-with-spyder.rst

```diff
@@ -13,15 +13,45 @@ Getting started
 
 Discover the basics of using the Spyder interface and get an introduction to its four main panes, along with a quick look at the others.
 
+* Find out different ways to open Spyder
+* Understand the key elements of Spyder’s interface
+* Learn more about Spyder’s four core panes
+* Explore the other default panes 
+
 .. youtube:: E2Dap5SfXkI
             :height: 360
             :width: 640
             :align: left
 
-* Find out different ways to open Spyder
-* Understand the key elements of Spyder’s interface
-* Learn more about Spyder’s four core panes
-* Explore the other default panes 
+.. rst-class:: dropdown-videos
+
+.. dropdown:: Transcript
+   :animate: fade-in
+
+   .. div:: scroll
+      
+      Hello everyone! I'm Juanita, and in this video I'm going to show you how to open Spyder and go over the basics of Spyder's interface. We will learn about Spyder's four panes that you'll likely be using most often, as well as briefly explore the others that are open by default. If you don't have Spyder installed and would like to follow along, you can download it at the link below this video.
+
+      The easiest way to open Spyder is by opening Anaconda Navigator and clicking on the Spyder application. In case you have an older version of Spyder in Anaconda, open the command line (or the Anaconda Prompt in the case of Windows) and type the commands:
+
+      .. code-block:: bash
+
+         conda update anaconda
+         conda install spyder=4
+
+      To launch Spyder without opening Navigator, open your command line and type ``spyder``. If you followed the :doc:`installation`, you should have everything necessary to open Spyder 4.
+
+      This is what Spyder 4 looks like in its default configuration, though you can thoroughly customize it, which we'll get to in a later tutorial. You can see that it is divided into three sections showing three different panes: the Editor, the IPython Console and the Help viewer. These three, along with the Variable Explorer, are the four core panes you'll work with the most in Spyder.
+ 
+      On the left we have the Editor, where you can open, edit and run files. Bottom right is the IPython Console, which you can use both interactively and to run your code in the Editor. It shows you which version of Python you are using. Above it, you'll find the Help pane, where you can get more information and documentation for any object in the  or  by pressing :kbd:`Ctrl-I` (or :kbd:`Cmd-I` on macOS). We'll see how to do this in our next video.
```

Comment:
```suggestion
      On the left we have the Editor, where you can open, edit and run files. Bottom right is the IPython Console, which you can use both interactively and to run your code in the Editor. It shows you which version of Python you are using. Above it, you'll find the Help pane, where you can get more information and documentation for any object in the Editor or Console by pressing :kbd:`Ctrl-I` (or :kbd:`Cmd-I` on macOS). We'll see how to do this in our next video.
```

My regex initially screwed up on this one, which I noticed and thought I went back and pasted the fixed version, but somehow the old version got pasted here...maybe I messed up pasting thye update in Google docs.